src/api/db/student_course_selection.py

```diff
@@ -0,0 +1,72 @@
+class student_course_selection:
+	def __init__(self, db_conn):
+		self.db_conn = db_conn
+
+	def add_selection(self, cid, sem, uid):
+		try:
+			sql = 	"""
+					INSERT INTO
+						student_course_selection (user_id, semester, course_id)
+					VALUES
+						(%s, %s, %s)
+					"""
+			resp, error = self.db_conn.execute(sql, [uid, sem, cid], False)
+		
+		except Exception as e:
+			return (False, e)
+
+		if error is not None:
```

Comment:
I don't think `error` is available at this part of the code.

Reply:
Moved